### DISCUSSION TITLE
获取百度令牌时 提示定向地址错误

### DISCUSSION THREAD

**Discussion by zhuchaojiebao** [AlistGo/alist#3513] [Q&A]
错误代码：redirect_uri_mismatch
错误信息：无效的重定向uri
详细描述：授权返回调整页面地址错误，具体内容请开发者参考“redirect uri”验证方式

**Comment by goodrabbit** [comment]
貌似作者更改了API的回调地址导致的。研究了一下，正确获得令牌的链接应该是这个：

https://openapi.baidu.com/oauth/2.0/authorize?response_type=code&client_id=iYCeC9g08h5vuP9UqvPHKKSVrKFXGa1v&redirect_uri=https://alist.nn.ci/tool/baidu/callback&scope=basic,netdisk&qrcode=1

![屏幕截图 2023-02-21 182552](https://user-images.githubusercontent.com/5088561/220319668-e89f299f-6363-4073-85ed-d3bfa464145a.jpg)

**Reply by zhuchaojiebao** [reply]
谢谢您的帮忙！
